.github/workflows/pr-deploy.yml

```diff
@@ -0,0 +1,49 @@
+name: PR Deploy
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '10.x'
+
+      - name: Cache dependencies
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      
+      - name: Install dependencies
+        run:  npm install
+        
+      - name: Compile CSS
+        run:  npm run css-compile
+        
+      - name: Copy Fonts
+        run:  cp -a dist/fonts/. guide/fonts
+
+      - name: Copy Images
+        run:  cp -a dist/images/. guide/images
```

Comment:
If we are copying these directories on build, should we remove/ignore the `guide/images/` directory from this PR?

Reply:
It has some additional images used in the guide. I could remove the duplicate files to reduce the repo size, but they'll need to be copied over for development.

Reply:
Take a look at the new version. I removed any duplicate/unused images/fonts from the guide and created npm scripts to copy these items over from dist if needed.